docs/reference/skeleton-based-action-recognition.md

```diff
@@ -719,27 +737,224 @@ Parameters:
 ```
 
-Download utility for various skeleton-based action recognition components. Downloads files depending on mode and
-saves them in the path provided. It supports downloading:
-1. the pretrained weights for stgcn, tagcn and stbln models. 
-2. a dataset containing one or more skeleton sequences and its labels.  
+Download utility for various skeleton-based action recognition components.
+Downloads files depending on mode and saves them in the path provided.
+It supports downloading:
+1. the pretrained weights for stgcn, tagcn and stbln models.
+2. a dataset containing one or more skeleton sequences and its labels.
+
+Parameters:
+
+- **path**: *str, default=None*\
+  Local path to save the files, defaults to self.parent_dir if None.
+- **mode**: *str, default="pretrained"*\
+  What file to download, can be one of "pretrained", "train_data", "val_data", "test_data"
+- **verbose**: *bool, default=False*\
+  Whether to print messages in the console.
+- **url**: *str, default=OpenDR FTP URL*\
+  URL of the FTP server.
+- **file_name**: *str*\
+  The name of the file containing the pretrained model.
+
+
+### Class CoSTGCNLearner
+Bases: `engine.learners.Learner`
+
+The *CoSTGCNLearner* class is an implementation of the proposed method CoSTGCN [[8]](#8) for Continual-Skeleton-based Human Action Recognition.
+It performs skeleton-based action recognition continuously in a frame-wise manner.
+The [CoSTGCNLearner](/src/opendr/perception/skeleton_based_action_recognition/continual_stgcn_learner.py) class has the following public methods:
+
+
+#### `CoSTGCNLearner` constructor
+```python
+CoSTGCNLearner(self, lr, iters, batch_size, optimizer, lr_schedule, backbone, network_head,
+               checkpoint_after_iter, checkpoint_load_iter, temp_path,
+               device, loss, weight_decay, momentun, drop_last, pin_memory, num_workers, seed,
+               num_classes, num_point, num_person, in_channels, graph_type, sequence_len
+               )
+```
+
+Constructor parameters:
+
+- **lr**: *float, default=0.001*\
+  Specifies the learning rate to be used during training.
+- **iters**: *int, default=10*\
+  Number of epochs to train for.
+- **batch_size**: *int, default=64*\
+  Specifies number of skeleton sequences to be bundled up in a batch during training.
+  This heavily affects memory usage, adjust according to your system.
+- **optimizer**: *str {'sgd', 'adam'}, default='adam'*\
+  Name of optimizer to use ("sgd" or "adam").
+- **lr_schedule**: *str, default=' '*
+  Specifies the learning rate scheduler.
+- **network_head**: *str, default='classification'*\
+  Head of network (only "classification" is currently available).
+- **checkpoint_after_iter**: *int, default=0*\
+  Unused parameter.
+- **checkpoint_load_iter**: *int, default=0*\
+  Unused parameter.
+- **temp_path**: *str, default=''*\
+  Path in which to store temporary files.
+- **device**: *{'cpu', 'cuda'}, default='cuda'*\
```

Comment:
`loss, weight_decay, momentum, drop_last, pin_memory, num_workers, seed, sequence_len` not documented

Reply:
Fixed in 26f18a31d86